CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # CHANGELOG
 
+## v0.37.8
+
+*July 1, 2024*
+
+This release reverts the API-breaking change to the `Mempool` interface introduced in the last patch
+release (v0.37.7) while still keeping the performance improvement added to the mempool. It also
+includes a minor fix to the RPC endpoints `/tx` and `/tx_search`.
+
+### BREAKING CHANGES
+
+- `[mempool]` Revert adding the method `PreUpdate()` to the `Mempool` interface, recently introduced
+  in the previous patch release. Its logic is now moved into the `Lock` method. With this change,
```

Comment:
Let's give the specific version numbers. I am not sure I know myself, please double check my suggestion

```suggestion
  in the previous patch release (v0.37.7). Its logic is now moved into the `Lock` method. With this change,
```